src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
There's a discrepancy between the file name used here (`star-active.svg`) and the file structure provided (`star_actived.png`). Ensure that the file names are consistent and correct. The file extension should also be `.png` as per the file structure.